src/ansys/aedt/core/application/analysis.py

```diff
@@ -2402,3 +2232,273 @@ def number_with_units(self, value, units=None):
+    def all(self):
+        """Create a dictionary with variables names associated to ``"All"``.
+
+        Returns
+        -------
+        dict
+            Dictionary of all variables with ``"All"`` value.
+
+        """
+        return {name: "All" for name in self.__variable_names()}
+
+    @property
+    def nominal(self):
+        """Create a dictionary with variables names associated to ``"Nominal"``.
+
+        Returns
+        -------
+        dict
+            Dictionary of all variables with ``"Nominal"`` value.
+        """
+        return {name: "Nominal" for name in self.__variable_names()}
+
+    @property
+    def nominal_values(self):
+        """All variables with nominal values.
+
+        Returns
+        -------
+        dict
+            Dictionary of nominal variations with values.
+
+        """
+        available_variables = self.__available_variables()
+        return {k: v.expression for k, v in list(available_variables.items())}
+
+    @property
+    def nominal_w_values_dict(self):
+        """Nominal independent with values in a dictionary.
+
+        Returns
+        -------
+        dict
+            Dictionary of nominal independent variations with values.
+
+        References
+        ----------
+        >>> oDesign.GetChildObject('Variables').GetChildNames
+        >>> oDesign.GetVariables
+        >>> oDesign.GetVariableValue
+        >>> oDesign.GetNominalVariation
+        """
+        warnings.warn(
+            "`nominal_w_values_dict` is deprecated. " "Use `nominal_values` method instead.", DeprecationWarning
+        )
+        families = {}
+        for k, v in list(self._app.variable_manager.independent_variables.items()):
+            families[k] = v.expression
+
+        return families
+
+    @property
+    def variables(self):
+        """Variables.
+
+        Returns
+        -------
+        list of str
+            List of names of independent variables.
+        """
+        warnings.warn(
+            "`variables` is deprecated. " "Use `variable_manager.independent_variable_names` method instead.",
+            DeprecationWarning,
+        )
+        return self._app.variable_manager.independent_variable_names
+
+    @property
+    def nominal_w_values(self):
+        """Nominal independent with values in a list.
+
+        Returns
+        -------
+        list
+            List of nominal independent variations with expressions.
+
+        References
+        ----------
+        >>> oDesign.GetChildObject('Variables').GetChildNames()
+        >>> oDesign.GetVariables
+        >>> oDesign.GetVariableValue
+        >>> oDesign.GetNominalVariation
+        """
+        warnings.warn("`nominal_w_values` is deprecated. " "Use `nominal_values` method instead.", DeprecationWarning)
+        families = []
+        for k, v in list(self._app.variable_manager.independent_variables.items()):
+            families.append(k + ":=")
+            families.append([v.expression])
+        return families
```

Comment:
```suggestion
        families = [
    f"{k}:=",
    [v.expression]
    for k, v in self._app.variable_manager.independent_variables.items()
]
return families
```
@Samuelopez-ansys same here.